src/interface/meta.thrift

```diff
@@ -96,9 +95,9 @@ struct TagItem {
     4: common.Schema        schema,
 }
 
-struct AlterSchemaItem {
-    1: AlterSchemaOp        op,
-    2: common.Schema        schema,
+struct AlterSchemaOption {
```

Comment:
Ditto

Reply:
I think this way is clearer.
We only modify the column in schem here, but cannot modify the propertyin schema.

Reply:
Why this changes?   Just rename the struct?

Reply:
Change it for my better understanding when using my first method.
Now, it has already corrected.